lib/model.js

```diff
@@ -110,9 +110,13 @@ internals.Validator = class SchwiftyValidator extends Objection.Validator {
         }
 
         const validation = ctx.joiSchema.validate(json);
+        //console.log('JUST VALIDATED', validation);
```

Comment:
Can you nix this?

Reply:
daah shit. yup, sorry about that. sloppy sloppy